File: src/main/java/io/github/chaosdave34/kitpvp/kits/impl/elytra/TeleporterKit.java
package io.github.chaosdave34.kitpvp.kits.impl.elytra;

import io.github.chaosdave34.kitpvp.kits.ElytraKit;
import org.bukkit.Material;
import org.bukkit.enchantments.Enchantment;
import org.bukkit.inventory.ItemStack;

public class TeleporterKit extends ElytraKit {
    public TeleporterKit() {
        super("elytra_teleporter", "Teleporter", Material.ENDER_PEARL);
    }

    @Override
    public ItemStack[] getInventoryContent() {
        ItemStack bow = new ItemStack(Material.BOW);
        bow.addEnchantment(Enchantment.ARROW_INFINITE, 1);
        bow.addEnchantment(Enchantment.ARROW_DAMAGE, 3);

        return new ItemStack[]{
                bow,
        };
    }

    @Override
    public ItemStack[] getKillRewards() {
        return new ItemStack[]{
                new ItemStack(Material.ENDER_PEARL, 5)
        };
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/kits/impl/elytra/TeleporterKit.kt
package io.github.chaosdave34.kitpvp.kits.impl.elytra

import io.github.chaosdave34.kitpvp.kits.ElytraKit
import org.bukkit.Material
import org.bukkit.enchantments.Enchantment
import org.bukkit.inventory.ItemStack

class TeleporterKit : ElytraKit("elytra_teleporter", "Teleporter", Material.ENDER_PEARL) {
    override fun getInventoryContent(): Array<ItemStack?> {

        val bow = ItemStack(Material.BOW)
        bow.addEnchantment(Enchantment.INFINITY, 1)
        bow.addEnchantment(Enchantment.POWER, 3)

        return arrayOf(
            bow,
        )
    }

    override fun getKillRewards(): Array<ItemStack> {
        return arrayOf(
            ItemStack(Material.ENDER_PEARL, 5)
        )
    }
}